plugins/services/src/js/components/SDKPlansScreen.tsx

```diff
@@ -0,0 +1,65 @@
+import * as React from "react";
+
+import { Service } from "#PLUGINS/services/src/js/types/Service";
+import SDKPlanTable from "#PLUGINS/services/src/js/components/SDKPlanTable";
+import { formatServicePlanStatus } from "#PLUGINS/services/src/js/types/ServicePlanStatus";
+import { ServicePlan } from "#PLUGINS/services/src/js/types/ServicePlan";
+
+export interface SDKPlansScreenProps {
+  service: Service;
+  plan: string;
+  handleSelectPlan: (name: string) => void;
+}
+
+class SDKPlansScreen extends React.PureComponent<SDKPlansScreenProps, {}> {
+  constructor(props: SDKPlansScreenProps) {
+    super(props);
+
+    this.renderPlanSelect = this.renderPlanSelect.bind(this);
```

Comment:
we could go with `renderPlanSelect  = () => {}` kind of binding instead.

Reply:
you mean writing the class in the constructor?

Reply:
I had the following version of binding in mind:
```ts
class SDKPlansScreen extends React.PureComponent<SDKPlansScreenProps, {}> {
  private renderPlanSelect = (plans: ServicePlan[]) => {
    // ...
  }
}
```

Reply:
Which is preferred? I asked this when working in `ui-kit`, and we prefer `.bind(this)` in the constructor there.

Reply:
I don't have a strong opinion on that just now. I believe back then we didn't have any other option thus rolling with the `.bind` as we're moving to TS we could reconsider and pick a cleaner solution.